File: ARCADE/Intro/Level 2_3: MakeArrayConsecutive2.cs
    using System;
     
    public class Test
    {
    	public static int makeArrayConsecutive2(int[] statues){
            int max = 0, min = 20;
            for (int i = 0; i < statues.Length; i++){
                if (statues[i] > max) max = statues[i];
                if (statues[i] < min) min = statues[i];
            }
            return (max - min - (statues.Length - 1));
        }
    	public static void Main()
    	{
    		int [] statues = new int [] {6, 2, 3, 8};
    		Console.WriteLine(makeArrayConsecutive2(statues));
    	}
    }

File: ARCADE/Intro/Level 2_3: MakeArrayConsecutive2.pas
    program makeArrayConsecutive2;
    Type
    	ArrayInteger = Array of Integer;
     
    function makeArrayConsecutive2(statues: ArrayInteger): integer;
    var max, min, i : integer;
    begin
        max := 0; min := 20;
        for i := 0 to length(statues) - 1 do
            begin
                 if(statues[i] > max) then max := statues[i];
                 if(statues[i] < min) then min := statues[i];
            end;
        makeArrayConsecutive2 := max - min - (length(statues) - 1);
    end;
     
    var statues : ArrayInteger;
    begin
    	statues := ArrayInteger.Create(6, 2, 3, 8);
    	writeln(makeArrayConsecutive2(statues));
    end.
